submissions/kazeemErinfolami/htmlForms/contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
Self closing tags only end without a `/`. This should be `<meta charset="UTF-8">`